testgrid/metadata/job.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+// Started holds the started.json values of the build.
+type Started struct {
+	Timestamp   int64             `json:"timestamp"` // epoch seconds
+	RepoVersion string            `json:"repo-version"`
```

Comment:
This should not be first-class

Reply:
Ack

It exists right now. Can we debate how to represent the commit of the primary repo later?